readthedocs_theme/templates/includes/topnav.html

```diff
@@ -106,12 +106,12 @@
 {# Application links to log in #}
 {% macro menu_log_in(with_signup=False) %}
   <div class="header">Log in</div>
-  <a class="item" data-analytics="community-login" href="https://readthedocs.org/dashboard/">
+  <a class="item" data-analytics="community-login" href="https://app.readthedocs.org/dashboard/">
     <i class="fad fa-people-group primary icon"></i>
     Read the Docs Community
 
     <p class="ui mini grey text">
-      <code>https://readthedocs.org</code>
+      <code>https://app.readthedocs.org</code>
```

Comment:
I think we might not want to change this one, since it's more conceptually "the .org site" and not the specific URL.

Reply:
I wonder if we want to remove the `https://` here, to make it more concrete that it's conceptual?